lisa/util/process.py

```diff
@@ -396,6 +402,15 @@ def _filter_profile_error(self, raw_input: str) -> str:
             )
         return raw_input
 
+    def _check_if_need_input_password(self, raw_input: str) -> None:
+        # Check if the stdout contains "[sudo] password for .*: " and
+        # "sudo: timed out reading password" strings. If so, raise exception
+        if re.search(REQUIRE_INPUT_PASSWORD_PATTERN, raw_input):
+            raise LisaException(
```

Comment:
is it RequireUserPasswordException?

Reply:
Yes, thank you Lili for reminding me. Have changed.